### DISCUSSION TITLE
Return in response_model additional info (like Pydantic schema - field title, type etc.)

### DISCUSSION THREAD

**Discussion by alispa** [fastapi/fastapi#8437] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel, Field
from typing import List

app = FastAPI()

class Item(BaseModel):
    name: str = Field(title="Product name")
    description: str = Field(title="Product description")

@app.get("/items", response_model=List[Item])
async def get_items():
    return get_all_items_from_db()
```

### Description

* I call the enpoint `/items`
* it has `response_model=List[Item]`
* it returns:
```
[
  {name: "apple", description: "amazing"},
  {name: "banana", description: "also amazing"},
]
```
* I am trying at the frontend to use field titles instead of the keys (`name`, `description`) that I now get.

### Wanted Solution

1) I would like `response_model={"fields_info": Item.schema(), "values": List[Item]}`
EXAMPLE:
```
{
  fields_info: {
    name: {title: "Product name", type: "string"},
    description: {title: "Product description", type: "string"}
  },
  values: [
    {name: "apple", description: "amazing"},
    {name: "banana", description: "also amazing"},
  ]
}
```

2) To make this response schema visible in the docs

### Wanted Code

```python
@app.get("/items", response_model={"fields_info": Item.schema(), "values": List[Item]})
async def get_items():
    return get_all_items_from_db()
```

### Operating System

Windows

### FastAPI Version

0.70.0

### Python Version

3.10.0

**Comment by YuriiMotov** [comment, marked as answer]
The frontend should request openapi schema using "/openapi.json" path and take titles from it:
```
  "components": {
    "schemas": {
      "Item": {
        "properties": {
          "name": {
            "type": "string",
            "title": "Product name"
          },
          "description": {
            "type": "string",
            "title": "Product description"
          }
        },
...
```
